content/participate/report-issue/redirect.html.haml

```diff
@@ -0,0 +1,102 @@
+---
+layout: simplepage
+title: Report an issue in a plugin
+section: participate
+noanchors: true
+---
+
+:css
+    .bg-img {
+        background-size: cover;
+        min-height: 20px;
+        background-position: center;
+    }
+    .bg-img-bug {
+        background-color: red;
+    }
+    .btn-bug {
+        border: 1px solid red;
+        color: 
+    }
+    .btn-bug:hover {
+        background-color: red;
+        color: white;
+    }
+
+
+    .bg-img-rfe {
+        background-color: #99f;
+    }
+    .btn-rfe {
+        border: 1px solid #99f;
+    }
+    .btn-rfe:hover {
+        background-color: #99f;
+        color: white;
+    }
+
+
+    .bg-img-security {
+        background-color: #fa0;
+    }
+    .btn-security {
+        border: 1px solid #fa0;
+    }
+    .btn-security:hover {
+        background-color: #fa0;
+        color: white;
+    }
+
+
+    .bl {
+        position: absolute;
+        bottom: 8.6%;
+        left: 4.4%;
+    }
+    .p1 {
+        margin-bottom: 22%;
+    }
+
+%p
+    Choose the kind of issue you'd like to report below.
+    %strong
+        Please be sure to to follow the 
+        %a{:href => '/participate/report-issue/' }
+            advice on how to report great issues.
+
+.grid-container
+    .card
+        .bg-img.bg-img-bug
+        .content
+            %h3
+                Bug
+            %p.p1
+                Report that something is broken.
+            %a.btn.btn-bug.bl#link-bug{:href => 'https://issues.jenkins.io/secure/CreateIssueDetails!init.jspa?pid=10172&issuetype=1&priority=4' }
+                Continue
+    .card
+        .bg-img.bg-img-rfe
+        .content
+            %h3
+                Improvement
+            %p.p1
+                Request a new feature or improvement.
+            %a.btn.btn-rfe.bl#link-rfe{:href => 'https://issues.jenkins.io/secure/CreateIssueDetails!init.jspa?pid=10172&issuetype=2&priority=4' }
+                Continue
+    .card
+        .bg-img.bg-img-security
+        .content
+            %h3
+                Security Issue
+            %p.p1
+                Report a security issue privately.
+            %a.btn.btn-security.bl{:href => 'https://issues.jenkins.io/secure/CreateIssueDetails!init.jspa?pid=10180&issuetype=10103&components=17329' }
```

Comment:
I expected that link to redirect to /security/reporting/ instead, to provide more details.

No strong opinion.

Reply:
Either would work, it's probably useful to have alike link targets.

I thought a reference to that page would make a useful "Learn more" link in the text, but haven't gotten around to adding it.